src/snapshotter/snapshotter.ts

```diff
@@ -74,6 +74,13 @@ set -e
 # Remove /var/lib/warpbuild-agentd/settings.json
 sudo rm /var/lib/warpbuild-agentd/settings.json
 
+# This command forces a write of all buffered I/O data to the disks. 
+echo "Flushing file system buffers..."
+sync
+    
+# Pause for a few seconds to ensure all I/O operations are complete
+sleep 5
```

Comment:
Why this arbitrary waiting period? Don't we have some way to check if all the I/O operations were finished?

Reply:
sync automatically does this, this should not be required. Removing this.

Reply:
removed